com.unity.render-pipelines.universal/CHANGELOG.md

```diff
@@ -6,8 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [10.4.0] - 2020-01-26
 
-Version Updated
-The version number for this package has increased due to a version update of a related graphics package.
+### Changed
```

Comment:
is this the correct spot for this changelog modification? under 10.4.0

Reply:
Yes